app/src/main/java/org/fossasia/susi/ai/forgotpassword/ForgotPasswordActivity.kt

```diff
@@ -41,6 +41,12 @@ class ForgotPasswordActivity : AppCompatActivity(), IForgotPasswordView {
             }
         }
 
+        var bundle = intent.extras;
+        if (bundle != null) {
+            var string = bundle.getString("email")
```

Comment:
There is no guarantee that email will be in this bundle, this should be nullable